backend/src/domain/Post.js

```diff
@@ -0,0 +1,16 @@
+class Post {
+
+  constructor(title) {
+    this.title = title;
+    this.votes = 0;
+  }
+
+  static async save(session, post) {
+    const { records: postRecords } = await session.writeTransaction((tx) =>
+      tx.run("CREATE (p: Post {title: $title, votes: 0}) RETURN p", post)
+    )
```

Comment:
:star: Data created in mutations are persisted and your queries and mutations still work as expected.